### DISCUSSION TITLE
What is the difference between Vanilla Arch and Black Arch (apart from community repo)?

### DISCUSSION THREAD

**Discussion by md3rn** [BlackArch/blackarch#4393] [Q&A] [answered]
I **don't** wanna be annoying , I just wanna know if there is any benefit in installing Black Arch over just getting the repos on my Arch installation.

**Comment by noraj** [comment]
I, personally, always recommend installing BA repo on top of an AL installation and then install the BA packages you need.

**Reply by md3rn** [reply]
But are there any differences? (except for repos and custom install script and graphic manager/window manger config)

**Reply by noraj** [reply]
It depends on what ISO you are using?

You can look at the source here: https://github.com/BlackArch/blackarch-iso

The main diff is on a true AL system you read the wiki, and you make you own choice and install manually, whereas as with BA ISOs you have limited choices and must follow pre-made steps.

I guess you don't care about details like `/etc/os-release` reporting AL or BA. Also depending on the ISO, some packages will be pre-installed.

**Reply by md3rn** [reply]
> It depends on what ISO you are using?
> 
> You can look at the source here: https://github.com/BlackArch/blackarch-iso
> 
> The main diff is on a true AL system you read the wiki, and you make you own choice and install manually, whereas as with BA ISOs you have limited choices and must follow pre-made steps.
> 
> I guess you don't care about details like `/etc/os-release` reporting AL or BA. Also depending on the ISO, some packages will be pre-installed.

To sum up, would I be correct in concluding that, there are no benefits to using BA over AL (except for the installation process and some preinstalled DE's and WM's)?

**Reply by noraj** [reply, marked as answer]
> To sum up, would I be correct in concluding that, there are no benefits to using BA over AL (except for the installation process and some preinstalled DE's and WM's)?

It doesn't make much sens to compare AL and BA, but I guess you rather mean:

```diff
-there are no benefits to using BA over AL
+there are no benefits to using BA ISOs over BA repo on top AL
```

If that's the question, then the answer is YES. BA ISOs often suffer bugs and issues. BA on top of AL will offer a better experience if one is not afraid of installing AL. BA ISOs are there for some edge cases, quick PoC or unexperienced users.
